src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js

```diff
@@ -591,9 +589,9 @@ const ClinicalVariableCard: React.ComponentType<IVariableCardProps> = ({
 }) => {
   const tableData = variable.active_chart === 'box'
     ? getBoxTableData(dataValues)
-    : getCategoricalTableData(
-      customBins,
-      getBucketRangesAndFilters,
+    : getTableData(
```

Comment:
renaming things to specify if they're used in continuous or categorical binning, or both